nil/common/logging/logger.go

```diff
@@ -139,6 +139,14 @@ func NewLogger(component string) zerolog.Logger {
 		Logger()
 }
 
+func NewMetricsLogger(component, table string) zerolog.Logger {
+	return NewLogger(component).With().
+		Bool("store_to_clickhouse", true).
+		Str("database", "metrics").
+		Str("table", table).
```

Comment:
I'm not sure you solve your problem properly.
Don't you need separate type of logger with custom exporter directly to clickhouse?

Reply:
wdym? exactly this way of logging to CH was proposed by @Zerg1996 in another review earlier

Reply:
Oh... I see...

https://github.com/NilFoundation/nil/blob/f23f70942199fc7d64b72aba79a8386bdd0aad97/nil/services/journald_forwarder/main.go#L222